src/main/java/com/netflix/imflibrary/st0377/HeaderPartition.java

```diff
@@ -194,9 +194,13 @@ public HeaderPartition(ByteProvider byteProvider, long byteOffset, long maxParti
                             prefaceSetCount));
         }
 
-        if (imfErrorLogger.getNumberOfErrors() > numErrors)
+        if (imfErrorLogger.getNumberOfErrors() > 0)
         {
-            throw new MXFException(String.format("%d errors encountered when reading header partition", imfErrorLogger.getNumberOfErrors() - numErrors));
```

Comment:
Wouldnt it make sense for the HeaderPartition Object to flag an exception only if parsing of led to accumulation of more errors in the error logger? That is, perhaps we should use the following logic:

"if (imfErrorLogger.getNumberOfErrors() > numErrors)"